schema/kind.go

```diff
@@ -72,10 +72,10 @@ const (
 	// Float64Kind is a float64 type and values of this type must be of the go type float64.
 	Float64Kind
 
-	// Bech32AddressKind is a bech32 address type and values of this type must be of the go type []byte.
-	// Fields of this type are expected to set the AddressPrefix field in the field definition to the
-	// bech32 address prefix so that indexers can properly convert them to strings.
-	Bech32AddressKind
+	// AddressKind represents an account address and must be of type []byte. Addresses usually have a
+	// human-readable rendering, such as bech32, and tooling should provide a way for apps to define a
+	// string encoder for friendly user-facing display.
+	AddressKind
```

Comment:
Most user facing addresses are simply account addresses, but in staking we do have validator addresses exposed to users. Two approaches to handling this are:

1. add a separate `ValidatorAddressKind` - this would mirror x/tx which supports only account and validator address
2. allow some open ended list of address types as `Field` options - this could be useful if we expect apps will expose other address types because validator and account addresses in queries

Reply:
Anyone have an opinion here? Kind of leaning towards 1. I don't think it's great practice to proliferate lots of bech32 prefixes in one app. They're all account addresses in the end